.github/workflows/cache.yml

```diff
@@ -0,0 +1,78 @@
+name: test cache
+
+on:
+  workflow_call:
+    outputs:
+      path:
+        value: ${{ jobs.cache.outputs.path }}
+      key:
+        value: ${{ jobs.cache.outputs.key }}
+  workflow_dispatch:
+  schedule:
+    # Weekly Monday 9AM
+    - cron: "0 9 * * 1"
```

Comment:
Should we maybe have this run during the night, at say midnight?

Reply:
```suggestion
    - cron: "0 0 * * 1"
```